source/simulator/parameters.cc

```diff
@@ -1637,7 +1637,8 @@ namespace aspect
             }
           else
             {
-              const int error_code = Utilities::MPI::broadcast(mpi_communicator, error_code, /* root = */ 0);
```

Comment:
Oh, I had not realized that what happens here is that it uses the same variable on the right that is being declared on the left!